### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by stephanlinke87** [nVuln/homebridge-lg-thinq#34] [General]
Just a quick thank you for this plugin :) We do have a GBB62PZFFN refridgerator and from authentication to state detection and settings, this plugin works flawlessly. Great work! :)

**Comment by pg8wood** [comment]
+1 this plugin is incredible! Nice work 💯

**Reply by nVuln** [reply]
@stephanlinke87 @pg8wood thank you so much ❤️
